controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -22,12 +111,54 @@ void wfa_ca::handle_wfa_ca_message(
         return;
     }
 
-    auto command_tuple = request->command(0);
-    if (!std::get<0>(command_tuple)) {
-        LOG(ERROR) << "getting command entry has failed!";
-        return;
-    }
-    const char *command_ptr = (char *)(&std::get<1>(command_tuple));
+    const char *command_ptr = request->command();
```

Comment:
instead of creating 2 variables, just use one - `std::string command = request->command();`